app/scripts/modules/core/src/artifact/expectedArtifactSelector.component.ts

```diff
@@ -25,7 +25,7 @@ class ExpectedArtifactSelectorComponent implements IComponentOptions {
       <div class="container-fluid form-horizontal">
         <ng-form name="artifact">
           <div class="form-group">
-            <label class="col-md-3 sm-label-right">Expected Artifact</label>
+            <label class="col-md-3 sm-label-right">Expected Artifact <help-field key="kubernetes.manifest.expectedArtifact"/></label>
```

Comment:
Same comment here about parameters

Reply:
Actually, as long as the text isn't k8s specific I would just namespace these tooltips differently.